src/custom/utils/appData.ts

```diff
@@ -12,11 +12,14 @@ export type BuildAppDataParams = {
 export async function buildAppData({ chainId, slippageBips, referrerAccount, appCode }: BuildAppDataParams) {
   const sdk = COW_SDK[chainId]
 
-  const referrerParams = referrerAccount ? { address: referrerAccount } : undefined
+  const referrerParams =
+    referrerAccount && chainId === SupportedChainId.MAINNET ? { address: referrerAccount } : undefined
+
+  const quoteParams = { slippageBips }
 
   const doc = sdk.metadataApi.generateAppDataDoc({
     appDataParams: { appCode, environment: environmentName },
-    metadataParams: { referrerParams, quoteParams: { slippageBips } },
+    metadataParams: { referrerParams, quoteParams },
```

Comment:
Hmm weird. This change and David's above were already in `develop`

![Screen Shot 2022-07-27 at 10 15 48](https://user-images.githubusercontent.com/43217/181210669-cb4fb09c-fbce-49b5-812e-3828cdcb49fe.png)

🤔 🤷